enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -94,8 +94,9 @@ if (hasPremiumFeature("whitelabel")) {
     ...sections,
   }));
 
-  PLUGIN_APP_INIT_FUCTIONS.push(({ root }) => {
+  PLUGIN_APP_INIT_FUCTIONS.push(() => {
     updateColors();
+    MetabaseSettings.on("application-colors", updateColors);
```

Comment:
👍 Now the color updates without having to refresh the page.

I, however, notice a small problem when reverting the color back to the default color. Basically, when you delete the non-default color the setting is saved with default color, but the UI still keep the previous color that has been set.
https://www.loom.com/share/ec45fc338e764b0f80b5998893ecc979

Reply:
Fixed, please take a look